index.js

```diff
@@ -0,0 +1,35 @@
+document.addEventListener("DOMContentLoaded", ()=>{
+    let select = document.querySelector("select")
+    let url = "https://ghibliapi.herokuapp.com/films"
+    const getMovieNames = async () =>{
+        try {
+            let res = await axios.get(url)
+            let movies = res.data
+            movies.forEach(movie=>{
+                let option = document.createElement("option")
+                option.innerText=movie.title
+                select.appendChild(option)
+
+            })
+        }catch(err){
+            console.log(err)
+            debugger
+        }
+    const displayMovieInfo = async (url) =>{
+        try{
+            let res = await axios.get(url)
+            let info = res.data;
+            let descpt = document.qu
```

Comment:
Much like your `getMovieNames` function in here you have access to all the information you need here. try to console log what you're getting back to move forward on this issue.